### DISCUSSION TITLE
Not getting datasets to process on 2 of 4 computers

### DISCUSSION THREAD

**Discussion by PappaSmurf** [BOINC/boinc#5879] [Q&A]
**Describe the bug**

**Steps To Reproduce**
1. 
2. 

**Expected behavior**

**Screenshots**

**System Information**
 - OS: 
 - BOINC Version: 

**Additional context**

**Comment by PappaSmurf** [comment]
I have not run BOINC in several years and have updated the software to the latest version on 4 different computers.  When I attempt to dowload new datasets to process only two of the four machines download data.  I am currently running the following projects.

1.  Rosetta@home
2.  World Community Grid

When I press update within BOINC it goes out to look for data but does not download any.

How do I get the other two computers working?

Mark Moss

OS Version: Windows 64bit 19045.5073
BPINC Manager Version 8.0.2(x64)

**Comment by PappaSmurf** [comment]
I did not mean to close this issue, I am still having this issue.

**Comment by AenBleidd** [comment]
What is said on the BOINC Manager Event log on the machines that didn't receive any work?

**Comment by PappaSmurf** [comment]
One PC says the following

 

 

11/2/2024 2:04:08 PM | ***@***.*** | update requested by user

11/2/2024 2:04:12 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 2:04:12 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 2:04:13 PM | ***@***.*** | Scheduler request completed

11/2/2024 2:04:13 PM | ***@***.*** | Project requested delay of 31 seconds

11/2/2024 2:04:38 PM | ***@***.*** | project suspended by user

11/2/2024 2:04:41 PM | ***@***.*** | project resumed by user

11/2/2024 2:16:22 PM | ***@***.*** | update requested by user

11/2/2024 2:16:22 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 2:16:22 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 2:16:23 PM | ***@***.*** | Scheduler request completed

11/2/2024 2:16:23 PM | ***@***.*** | Project requested delay of 31 seconds

11/2/2024 2:18:10 PM | ***@***.*** | update requested by user

11/2/2024 2:18:15 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 2:18:15 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 2:18:16 PM | ***@***.*** | Scheduler request completed

11/2/2024 2:18:16 PM | ***@***.*** | Project requested delay of 31 seconds

11/2/2024 2:22:02 PM | ***@***.*** | update requested by user

11/2/2024 2:22:05 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 2:22:05 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 2:22:06 PM | ***@***.*** | Scheduler request completed

11/2/2024 2:22:06 PM | ***@***.*** | Project requested delay of 31 seconds

11/2/2024 2:24:45 PM | ***@***.*** | update requested by user

11/2/2024 2:24:48 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 2:24:48 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 2:24:49 PM | ***@***.*** | Scheduler request completed

11/2/2024 2:24:49 PM | ***@***.*** | Project requested delay of 31 seconds

11/2/2024 5:06:40 PM | World Community Grid | update requested by user

11/2/2024 5:06:45 PM | World Community Grid | Sending scheduler request: Requested by user.

11/2/2024 5:06:45 PM | World Community Grid | Requesting new tasks for CPU

11/2/2024 5:06:46 PM | World Community Grid | Scheduler request completed: got 0 new tasks

11/2/2024 5:06:46 PM | World Community Grid | No tasks sent

11/2/2024 5:06:46 PM | World Community Grid | No tasks are available for Africa Rainfall Project

11/2/2024 5:06:46 PM | World Community Grid | No tasks are available for the applications you have selected.

11/2/2024 5:06:46 PM | World Community Grid | Tasks for NVIDIA GPU are available, but your preferences are set to not accept them

11/2/2024 5:06:46 PM | World Community Grid | Tasks for AMD/ATI GPU are available, but your preferences are set to not accept them

11/2/2024 5:06:46 PM | World Community Grid | Tasks for Intel GPU are available, but your preferences are set to not accept them

11/2/2024 5:06:46 PM | World Community Grid | Project requested delay of 121 seconds

11/2/2024 5:06:57 PM | ***@***.*** | update requested by user

11/2/2024 5:06:57 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 5:06:57 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 5:06:58 PM | ***@***.*** | Scheduler request completed

11/2/2024 5:06:58 PM | ***@***.*** | Project requested delay of 31 seconds

11/2/2024 5:07:46 PM | ***@***.*** | update requested by user

11/2/2024 5:07:49 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 5:07:49 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 5:07:50 PM | ***@***.*** | Scheduler request completed

11/2/2024 5:07:50 PM | ***@***.*** | Project requested delay of 31 seconds

11/2/2024 5:08:45 PM | ***@***.*** | update requested by user

11/2/2024 5:08:46 PM | ***@***.*** | Sending scheduler request: Requested by user.

11/2/2024 5:08:46 PM | ***@***.*** | Not requesting tasks: don't need ()

11/2/2024 5:08:48 PM | ***@***.*** | Scheduler request completed

11/2/2024 5:08:48 PM | ***@***.*** | Project requested delay of 31 seconds

 

Mark Moss

**Comment by AenBleidd** [comment]
Basically  this is the answer:

```
11/2/2024 5:06:46 PM | World Community Grid | No tasks sent

11/2/2024 5:06:46 PM | World Community Grid | No tasks are available for Africa Rainfall Project

11/2/2024 5:06:46 PM | World Community Grid | No tasks are available for the applications you have selected.

11/2/2024 5:06:46 PM | World Community Grid | Tasks for NVIDIA GPU are available, but your preferences are set to not accept them

11/2/2024 5:06:46 PM | World Community Grid | Tasks for AMD/ATI GPU are available, but your preferences are set to not accept them

11/2/2024 5:06:46 PM | World Community Grid | Tasks for Intel GPU are available, but your preferences are set to not accept them

```
If you have GPUs available you should allow them in the Project's preferences on their website.
And just FYI: WCG doesn't have a consistent amount of available tasks nowadays.
You should connect to some other projects if you want.
